src/loaders/tiff/index.ts

```diff
@@ -20,8 +23,10 @@ interface OmeTiffOptions extends TiffOptions {
   images?: 'first' | 'all';
 }
 
+export type MultiTiffOptions = OmeTiffOptions;
```

Comment:
Sorry I didn't notice this earlier, but a lot of these options don't actually correspond to the `MultiTiiff`. I think probably only `pool` is used in the function, so probably

```typescript
export interface MultiTiffOptions {
	pool?: boolean;
}
```